.github/workflows/deploy-windows.yml

```diff
@@ -0,0 +1,83 @@
+name: deploy-windows
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
@lu-zero Are we planning to use the "v" prefix for future releases?

Reply:
We should decide, probably it is a good idea.

Reply:
I started using `p{date}` and `v{version}`.